apps/functions/applications-migration/project-updates-migration/src/project-updates-migration.js

```diff
@@ -1,34 +1,34 @@
 import { chunk as chunkArray } from 'lodash-es';
-import { QueryTypes, Sequelize } from 'sequelize';
-import { loadWordpressConfig } from '../../common/config.js';
 import { makePostRequest } from '../../common/back-office-api-client.js';
+import { executeSequelizeQuery } from './execute-sequelize-query.js';
 
 const MAX_BODY_ITEMS_LENGTH = 100;
 
-const config = loadWordpressConfig();
-const { username, password, database, host, port, dialect } = config.wordpressDatabase;
-
-const sequelize = new Sequelize(database, username, password, {
-	host,
-	port: Number(port),
-	// @ts-ignore
-	dialect
-});
-
 /**
  * Handle an HTTP trigger/request to run the migration
  *
  * @param {import('@azure/functions').Logger} log
  * @param {string[]} caseReferences
+ * @param {boolean} isWelshCase
  */
-export const migrateProjectUpdates = async (log, caseReferences) => {
+export const migrateProjectUpdates = async (log, caseReferences, isWelshCase) => {
 	log.info(`Migrating ${caseReferences.length} Cases`);
 
 	for (const caseReference of caseReferences) {
 		try {
 			log.info(`Migrating project updates and subscriptions for case ${caseReference}`);
 
-			const updates = await getProjectUpdates(log, caseReference);
+			const englishUpdates = await getProjectUpdates(log, caseReference);
+			const welshUpdates = isWelshCase ? await getProjectUpdatesWelsh(log, caseReference) : [];
+			if (isWelshCase && englishUpdates.length !== welshUpdates.length) {
+				throw Error('Update entity count for English and Welsh do not match');
+			}
+
+			const updates = englishUpdates.map((update, i) => ({
```

Comment:
Wouldnt this run even if welshUpdates is false? Think all the welsh stuff might need to be in an if block instead of a the ternary operator.